notifier/test/main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	// Works:
+	// "github.com/getlantern/systray"
+
+	// Does not work:
+	"fyne.io/systray"
+
+	"github.com/safing/portmaster-ui/notifier/icons"
+)
+
+func main() {
+	systray.SetIcon(icons.GreenPNG)
+	systray.Run(onReady, nil)
+}
+
+func onReady() {
```

Comment:
It looks like this `onReady` callback is blocking... 
It should return (so the for should run in a goroutine) then the app will continue.

Reply:
I've responded in the issue thread here: https://github.com/fyne-io/systray/issues/3#issuecomment-1110690810